tables/th-g0.utb

```diff
@@ -0,0 +1,178 @@
+# liblouis: Thai Current Braille System.
+#
+#-index-name: Thai
+#-display-name: Thai braille
+#
+#+language:th
+#+region:th-TH
+#+type:literary
+#+system:thai
+#+contraction: no
+#+grade: 0
+#+dots: 6
+# Marked as "direction:forward" by thai braille funds
+# as tests only run forward.
+#+direction:forward
```

Comment:
It seems to me that this table can also be used for backward translation, since it has the `include` rule at the bottom, and since the YAML file for this table contains backward tests.